app/counselor/views.py

```diff
@@ -153,15 +153,15 @@ def checklist():
         #create new checklist item from form data
         new_item = ChecklistItem(
                     text=form.item_text.data,
-                    assignee_id=current_user.id,
+                    assignee_id=0, #denotes counselor-created item
                     creator_role_id=3)
```

Comment:
Hmm this is hard because a counselor could have created a checklist item for every student, in which case we'd want all counselors to see it. Not actually sure how to handle this.